ansible/tasks/postgres-extensions/27-pg_repack.yml

```diff
@@ -13,7 +13,7 @@
    git:
      repo: https://github.com/reorg/pg_repack.git
      dest: /tmp/pg_repack
-     version: "ver_{{ pg_repack_release }}"
+     version: "{{ pg_repack_commit }}"
```

Comment:
Q: Is this ansible file still used for postgres extensions? I thought we only do the Nix change nowadays

Reply:
Thanks for the review. Indeed, I wasn't sure if it was necessary. I reverted that change. I also bumped postgres release, it seems it is necessary to do on bumping an extension version.